### PR TITLE
Add the Compatibility Schema of IntelliJ

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to add the compatibility schema mentioned in [1] to [2].

[1] https://github.com/ballerina-platform/ballerina-lang/tree/master/tool-plugins/intellij#release-versions-schema
[2] https://ballerina.io/learn/tools-ides/intellij-plugin/

Fixes https://github.com/ballerina-platform/ballerina-www/issues/1475.

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md [hunk 1]
```diff
@@ -7,9 +7,15 @@ The IntelliJ Ballerina plugin provides the Ballerina development capabilities in
 - [Using the plugin](#using-the-plugin)
 - [Using the features of the plugin](#using-the-features-of-the-plugin)
 
-## Downloading IntelliJ IDEA 
+## Downloading IntelliJ IDEA
 
-Download the [IntelliJ IDEA editor](https://www.jetbrains.com/idea/download/).
+Download the [IntelliJ IDEA editor](https://www.jetbrains.com/idea/download/) (or update it) by selecting the version from the below table based on the compatibility with the Ballerina plugin version, which you will be using.
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]:
```suggestion
Please refer the table below and [Download IntelliJ IDE](https://www.jetbrains.com/idea/download/) (or update the exiting IDE) based on its compatibility with the  plugin version (which is same as the Ballerina distribution version), which you will be using.

```

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md [hunk 2]
```diff
@@ -7,9 +7,15 @@ The IntelliJ Ballerina plugin provides the Ballerina development capabilities in
 - [Using the plugin](#using-the-plugin)
 - [Using the features of the plugin](#using-the-features-of-the-plugin)
 
-## Downloading IntelliJ IDEA 
+## Downloading IntelliJ IDEA
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]:
```suggestion
## Prerequisites
```

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md [hunk 3]
```diff
@@ -7,9 +7,15 @@ The IntelliJ Ballerina plugin provides the Ballerina development capabilities in
...
 
-Download the [IntelliJ IDEA editor](https://www.jetbrains.com/idea/download/).
+Download the [IntelliJ IDEA editor](https://www.jetbrains.com/idea/download/) (or update it) by selecting the version from the below table based on the compatibility with the Ballerina plugin version, which you will be using.
+
+**Plugin Version**|**Platform Version Compatibility**
+:-----:|:-----:
+0.8.0 - 0.8.2|IntelliJ IDEA 2016.3 - 2016.4
+0.8.3 - 0.981.0|IntelliJ IDEA 2016.3+
+0.982.0+|IntelliJ IDEA 2017.3+
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]:
```suggestion
0.982.0 - 0.991.0|IntelliJ IDEA 2017.3+
0.991.1+ | IntelliJ IDEA 2018.3+
```

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md (hunk 2 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]: Fixed.

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md (hunk 3 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]: Fixed.

File: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md (hunk 1 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md]: Discussed and changed.

**Pull request merged**